### PR TITLE
#158519698 User Login feature/Endpoint

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
#### What does this PR do?
Add endpoint for user login
#### Description of Task to be completed?
- Add view function to for login endpoint
- Implement actual user login within the view function
```
- POST /auth/login user can Login using this endpoint
```
#### How should this be manually tested?
using your terminal,
- clone this repo
- `cd ride-my-way`
- set and activate your virtual environment
- `git checkout ft-user-login-158519698`
- Run `nosetests tests/testUserAccount.py`

**screenshots**
login endpoint
user login

**Pivotal tracker story **
[[User login 158519698]](https://www.pivotaltracker.com/story/show/158519698)

### CODE REVIEW

File: application/views/user_views.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            return {'message': 'We are unable to create your account at the moment.'}, 404
+
+# model for login
+model_login = api.model('Login', {'email': fields.String,
+                                  'password': fields.String})
+
+
+class UserLogin(Resource):
+
+    @api.doc('user accounts', responses={201: 'CREATED', 400: 'BAD REQUEST', 401:'INVALID CREDENTIALS',404: 'NOT FOUND'})
```
Comment (xcixor) [inline application/views/user_views.py]: This line is too long, remember the 79 characters cap

Comment (meshack-mbuvi) [reply application/views/user_views.py]: Let me look into it

Comment (meshack-mbuvi) [reply application/views/user_views.py]: Corrected

**Comment by LWanjiru** [comment]
One of your builds is failing

**Pull request merged**